tools/shared.py

```diff
@@ -2577,7 +2577,7 @@ def closure_compiler(filename, pretty=True):
       proc = run_process(args, stderr=PIPE, check=False)
       if proc.returncode != 0:
         sys.stderr.write(proc.stderr)
-        exit_with_error('closure compiler failed (rc: %d)', proc.returncode)
+        exit_with_error('closure compiler failed (rc: %d. the error message may be clearer with -g1)', proc.returncode)
```

Comment:
"if not g1:"?

Reply:
Good idea, added a check for whether the -g1 hint is needed or not.